solutions.sql

```diff
@@ -0,0 +1,84 @@
+Step 1
+CREATE DATABASE books;
+Step2
+\c books;
+Step3
+\i /Users/jisuyeo/Documents/ga/homworks/booktown/seed.sql
+\i /Users/jisuyeo/Documents/ga/homworks/booktown/schema.sql
+id |           name
+----+--------------------------
+  1 | King    Stephen
+  2 | Herbert Frank
+  3 | Clarke  Arthur C.
+  4 | Bourgeois       Paulette
+  5 | Brown   Margaret Wise
+  6 | Alcott  Louisa May
+  7 | Bianco  Margery Williams
+  8 | Hogarth Burne
+  9 | Lutz    Mark
+
+Q1
+SELECT books.title FROM 
+books JOIN authors ON books.authorId = authors.id
+WHERE authors.name ='Lutz	Mark';
+Q2
+SELECT books.title FROM
+books JOIN subjects ON books.subjectId = subjects.id
+WHERE subjects.name = 'Childrens Books';
+Q3
```

Comment:
Question 3 was get all subjects that a given author has written on.
Looks like you started on the right path by Selecting subject.name but you stopped at the FROM command.
FROM subjects would gave us access to the subjects. -- Your join statement was looking good too. It was almost there "JOIN books ON authors.id = books.authorId
    JOIN subjects ON subjects.id = books.subjectId"
let know if this helps


get all authors that have written books on a given subject